xmtp_mls/src/identity_updates.rs

```diff
@@ -285,9 +286,30 @@ where
         let inbox_id = self.inbox_id();
         let builder = SignatureRequestBuilder::new(inbox_id);
 
-        Ok(builder
+        let installation_public_key = self.identity().installation_keys.verifying_key();
+        let mut signature_request = builder
+            .add_association(
+                installation_public_key.as_bytes().to_vec().into(),
```

Comment:
This looks wrong. You should be adding with the installation public key as the existing member, and the `new_wallet_address` address as the new member

Reply:
Thank you